apps/src/templates/manageStudents/PasswordReset.jsx

```diff
@@ -36,11 +36,25 @@ class PasswordReset extends Component {
   };
 
   save = () => {
-    this.props.resetAction(this.state.input);
-
-    this.setState({
-      isResetting: false,
-      input: ''
+    $.ajax({
+      url: `/v2/students/${this.props.id}/update`,
+      method: 'POST',
+      contentType: 'application/json;charset=UTF-8',
+      data: JSON.stringify({
+        password: this.state.input,
+        editing_password: true,
+      }),
+    }).done((data) => {
+      console.log(data);
```

Comment:
Do we usually keep these console.logs in or were you just using them to debug?

Reply:
oops good catch